docker-compose.yml

```diff
@@ -0,0 +1,68 @@
+version: "3.5"
```

Comment:
I guess docker-compose repo should be included as submodule here, to make it nice.